src/Context/Auth-context/Auth-context.js

```diff
@@ -0,0 +1,30 @@
+import { createContext, useContext, useState } from "react";
+import axios from "axios";
+const authContext = createContext();
+const useAuth = () => useContext(authContext);
+
+const AuthProvider = ({ children }) => {
+  const [userData, setUserData] = useState([]);
```

Comment:
Make it ```object``` instead of array as in future you can optional chain it and the data response is also an object

Reply:
okkk  sure